flow/layers/opacity_layer.cc

```diff
@@ -83,8 +71,7 @@ void OpacityLayer::Paint(PaintContext& context) const {
   // Skia may clip the content with saveLayerBounds (although it's not a
   // guaranteed clip). So we have to provide a big enough saveLayerBounds. To do
   // so, we first remove the offset from paint bounds since it's already in the
-  // matrix. Then we round out the bounds because of our
-  // RasterCache::GetIntegralTransCTM optimization.
+  // matrix. Then we round out the bounds.
```

Comment:
Would it be easier to pass an offset to the Draw command above so that it doesn't get added to the transform? This would also avoid an extra save?

Reply:
That's probably true. Since a normal matrix save (instead of a layer save) is quite inexpensive, and the change adds more risk to this already quite risky PR, I'm going to try that in a future PR.

Reply:
Agreed.